### DISCUSSION TITLE
Questions on the 7% used route-sets

### DISCUSSION THREAD

**Discussion by SichangHe** [SichangHe/internet_route_verification#174] [General]
> Section 4 mentions that "only 7% of defined route-sets are actually used". I wonder
how many of these are stale data

#130 says 20% are before 2014 and 50% are after 2021, so it is somewhat stale but not quite.

> or if these could be objects that are directly 
refered in external systems, for example in PeeringDB (https://docs.peeringdb.com/howto/get-started-operator/)?

I don't know about this one, @cunha.

- [x] Mention `last-modified` are similar (not shown).

**Comment by cunha** [comment]
I think it's important to mention that RS-SETS being unused does *not* imply that they are stale. We may need to reinforce the point that they just aren't employed. We could show that RS-SETS are *newer* than AS-SETS and used *less often* to drive this home.
